source/educators/how-tos/open_response.rst

```diff
@@ -1,3 +1,516 @@
 Add an Open Response Assessment
-#################################
+###############################
 
+Creating an open response assessment (ORA) assignment is a multi-step process.
+This section covers each step in detail.
+
+In addition, see these other topics about different aspects of open response
+assessments.
+
+* Components of an open response assessment: Introduction to Open Response
+  Assessments
+* Viewing metrics and learner responses for released open response assessments:
+  Accessing Metrics for ORA Assignments
+* Actions you can take for an active ORA assignment: Managing Open Response
+  Assessment Assignments
```

Comment:
These are not yet links, right? I think I wouldn't have these in here until we can have the links